packages/firebase_crashlytics/example/lib/main.dart

```diff
@@ -17,7 +17,7 @@ void main() {
   // Pass all uncaught errors to Crashlytics.
   FlutterError.onError = Crashlytics.instance.recordFlutterError;
 
-  runZoned<Future<void>>(() {
+  runZoned<void>(() {
```

Comment:
Is this some Dart change?

Reply:
@kroikie was working on fixing this https://github.com/FirebaseExtended/flutterfire/pull/1589#event-2868300280.

I believe it's from the analysis changes.